test/hotspot/jtreg/compiler/debug/TraceIterativeGVN.java

```diff
@@ -23,7 +24,7 @@
 
 /*
  * @test
- *
+ * @requires vm.debug == true & vm.flavor == "server"
```

Comment:
Use `vm.compiler2.enabled` instead of `vm.flavor == "server"`

Reply:
Got! updated in new commit.